lib/label_printer/label/multiple_labels.rb

```diff
@@ -19,7 +19,7 @@ def create_labels
         end
       end
 
-      def count
+      def count # rubocop:disable Lint/DuplicateMethods
```

Comment:
ADVISORY: This is a bit of a code smell??

Reply:
Yeah I was forgot to come back and fix this, addressed in https://github.com/sanger/sequencescape/pull/4371/commits/ca3c651b25bab117af2f3be578435ba284b07997